<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorUnit</class>
 <widget class="QWidget" name="MotorUnit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEditMotorNum">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>4</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelMotionList_25">
    <property name="geometry">
     <rect>
      <x>833</x>
      <y>7</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Command</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMotorInputData">
    <property name="geometry">
     <rect>
      <x>676</x>
      <y>6</y>
      <width>59</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHMinus">
    <property name="geometry">
     <rect>
      <x>891</x>
      <y>6</y>
      <width>39</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>H-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHPlus">
    <property name="geometry">
     <rect>
      <x>932</x>
      <y>6</y>
      <width>39</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>H+</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>6</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReset">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>5</y>
      <width>39</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLPlus">
    <property name="geometry">
     <rect>
      <x>1014</x>
      <y>6</y>
      <width>39</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>L+</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMotionList_24">
    <property name="geometry">
     <rect>
      <x>783</x>
      <y>7</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOn">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>4</y>
      <width>29</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLMinus">
    <property name="geometry">
     <rect>
      <x>973</x>
      <y>6</y>
      <width>39</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>L-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPosFast">
    <property name="geometry">
     <rect>
      <x>504</x>
      <y>6</y>
      <width>29</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNegSlow">
    <property name="geometry">
     <rect>
      <x>424</x>
      <y>6</y>
      <width>29</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlus">
    <property name="geometry">
     <rect>
      <x>594</x>
      <y>6</y>
      <width>29</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMotionList_23">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>7</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelIndex">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>5</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOFF">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>5</y>
      <width>29</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMotionList_22">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>7</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Disabled</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMinus">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>6</y>
      <width>29</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMove">
    <property name="geometry">
     <rect>
      <x>733</x>
      <y>6</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPosSlow">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>6</y>
      <width>29</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNegFast">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>6</y>
      <width>29</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <zorder>pushButtonLMinus</zorder>
   <zorder>lineEditMotorNum</zorder>
   <zorder>labelMotionList_25</zorder>
   <zorder>lineEditMotorInputData</zorder>
   <zorder>pushButtonHMinus</zorder>
   <zorder>pushButtonHPlus</zorder>
   <zorder>doubleSpinBox</zorder>
   <zorder>pushButtonReset</zorder>
   <zorder>pushButtonLPlus</zorder>
   <zorder>labelMotionList_24</zorder>
   <zorder>pushButtonOn</zorder>
   <zorder>pushButtonPosFast</zorder>
   <zorder>pushButtonNegSlow</zorder>
   <zorder>pushButtonPlus</zorder>
   <zorder>labelMotionList_23</zorder>
   <zorder>labelIndex</zorder>
   <zorder>pushButtonOFF</zorder>
   <zorder>labelMotionList_22</zorder>
   <zorder>pushButtonMinus</zorder>
   <zorder>pushButtonMove</zorder>
   <zorder>pushButtonPosSlow</zorder>
   <zorder>pushButtonNegFast</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
